Utils/layouts/head.js

```diff
@@ -14,6 +13,23 @@ const AppHead = ({ path, t }) => {
     pageTitle = t("head.page.landing");
   }
 
+  const $ = "window";
+
+  const script = process.browser ? (
```

Comment:
Likewise I'm not convinced we need to stop this script being added server-side. I don't believe it runs anything on load. I believe also that the isomorphic nature of the app will cause problems whenever the server-side and hydrated client-side code differ.

Reply:
No that wasnt the solution, i found. I have removed the` const $ = window`